test/extern/ferguson/extern-record-proc-other-init.good

```diff
@@ -0,0 +1,5 @@
+extern-record-proc-other-init.chpl:24: In function 'testMystruct':
+extern-record-proc-other-init.chpl:26: error: unresolved call 'R.init()'
+extern-record-proc-other-init.chpl:3: note: this candidate did not match: R.init(arg: int(64))
+extern-record-proc-other-init.chpl:26: note: because call does not supply enough arguments
+extern-record-proc-other-init.chpl:3: note: it is missing a value for formal 'arg: int(64)'
```

Comment:
Shouldn't this be a "cannot default initialize variable" or some sort of split-init error?

The error message is not ideal. But if it is not easy to tackle I don't have a problem with this for now -- not the worst error message I saw.

Reply:
Yeah, I think improving this error message is nice future work but it's outside of the scope of this PR.